.github/workflows/turbo-deploy.yml

```
@@ -0,0 +1,218 @@
+
+      - name: Set dynamic secret names
+        run: |
+          PROJECT_UPPER=$(echo "${{ matrix.project }}" | tr '[:lower:]' '[:upper:]')
+          echo "APP_ENV_VARS_SECRET=${PROJECT_UPPER}_APP_ENV_VARS" >> $GITHUB_ENV
+
+      - name: Create environment file with fallback
+        run: |
+          # Check if project-specific env vars exist, fallback to default
+          PROJECT_ENV_VARS="${{ secrets[env.APP_ENV_VARS_SECRET] }}"
+          DEFAULT_ENV_VARS="${{ secrets.DEFAULT_APP_ENV_VARS }}"
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)